x/txfees/keeper/txfee_filters/arb_tx.go

```diff
@@ -81,79 +81,52 @@ func IsArbTxLoose(tx sdk.Tx) bool {
 }
 
 func isArbTxLooseAuthz(msg sdk.Msg, swapInDenom string, lpTypesSeen map[gammtypes.LiquidityChangeType]bool) (string, bool) {
-	if authzMsg, ok := msg.(*authztypes.MsgExec); ok {
-		msgs, _ := authzMsg.GetMessages()
-		for _, m := range msgs {
-			swapInDenom, isAuthz := isArbTxLooseAuthz(m, swapInDenom, lpTypesSeen)
+	switch m := msg.(type) {
```

Comment:
Go switch cases does not fall through, AFAIK we cannot achieve the same behaviour as the prior logic using if statements. 

I might be misunderstanding your proposed changed, if so can you elaborate?

Reply:
You are right they don't fall through. Still except the check for `gammtypes.LiquidityChangeMsg` we return `swapDenom, false` as default in the if statement. Meaning there is no "fall through" in the ifs chain.

Though it is true that `gammtypes.LiquidityChangeMsg` is special and should probably be handled outside of the switch case. wdyt?

Reply:
my thoughts are because it doesnt fall through, it should all be handled in if statements (outside switch cases)